### PR TITLE
Test add tests for ut rule - new scenarios for daily, weekly, monthly limits

### PR DESCRIPTION
**Pull request by ReniPaskaleva**
@venetahristova, @lambrevalora, please review.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const utc = {
 module.exports = () => ({
     // environments
     dev: {
+        sqlTest: true,
```
Comment (venetahristova) [inline config.js:17]: @kalinkrustev , should we add this or not?

File: test/seed/852-rule.rule.merge.yaml [hunk 2]
```diff
@@ -196,7 +229,7 @@ splitRange: # defaults splitName: fee, startAmountCurrency: USD, startAmount*: 0
   - {conditionName: Test by counterparty organization, percent: 3.4}
   - {conditionName: Test by counterparty type, percent: 3.5}
   - {conditionName: Test by counterparty kyc, percent: 3.6}
-  - {conditionName: Test Split Range, percent: 0.8}
+  - {conditionName: Test Split Range, percent: 0.8}
```
Comment (venetahristova) [inline test/seed/852-rule.rule.merge.yaml:232]: remove the space at the end of the row

File: test/seed/852-rule.rule.merge.yaml [hunk 3]
```diff
@@ -165,7 +199,6 @@ splitName: # defaults name: fee, tag: |fee|acquirer|
   - {conditionName: Test by start date, name: base, tag: "|issuer|"}
   - {conditionName: Test by end date, name: testFee}
   - {conditionName: Test by end date, name: base, tag: "|issuer|"}
-  - {conditionName: Test settlement, name: base, tag: "|issuer|"}
```
Comment (venetahristova) [inline test/seed/852-rule.rule.merge.yaml:168]: should we remove this rule?

File: test/seed/852-rule.rule.merge.yaml (hunk 2 above)
Comment (ReniPaskaleva) [reply test/seed/852-rule.rule.merge.yaml:232]: Done

File: test/seed/852-rule.rule.merge.yaml (hunk 3 above)
Comment (ReniPaskaleva) [reply test/seed/852-rule.rule.merge.yaml:168]: It was removed by mistake while resolving merge conflicts. I have added it again.

**Comment by kalinkrustev** [comment]
Моля, след като се оправят нещата по коментарите, този рекуест да се насочи към major/currencies, вместо към master.

**Pull request merged**